cornice/validators/_marshmallow.py

```diff
@@ -46,10 +49,14 @@ def _validator(request, schema=None, deserializer=None, **kwargs):
         if schema is None:
             return
 
-        schema = _instantiate_schema(schema)
+        # see if the user wants to set any keyword arguments for their schema
+        schema_kwargs = {}
+        if 'schema_kwargs' in kwargs:
+            schema_kwargs = kwargs.get('schema_kwargs', {})
```

Comment:
Sorry this code suggestion thing didn't work as I expected. You can remove the above lines

Reply:
Sorry, you lost me in this. The truth is that I accepted this change without seeing it too deeply, I thought it was in a different spot. Now I'm a bit confused, has it been merged with my code or not?

Reply:
Just that these 3 lines can be replaced by the last one

`schema_kwargs = kwargs.get('schema_kwargs', {})`

Reply:
Is it OK now? I'm not familiar with these github's features (merging suggested changes and editing files within github), that's why I'm asking. 

So, I've edited the file within github to only contain `schema_kwargs = kwargs.get('schema_kwargs', {})` and I suppose it's OK now, but I'm not sure I saw the checks to have run again.